django_freeradius/urls.py

```diff
@@ -2,6 +2,7 @@
 
 from .api import urls as api
 
+app_name = 'freeradius'
```

Comment:
mm.. I don't understand this.. why is it here?

Reply:
in django 1.11 is not problematic if not written, but if in django 2 this will cause error on namespace in include imported by django.conf.urls

Reply:
https://github.com/encode/django-rest-framework/pull/5609

Reply:
ok, thx for pointing it out